### PR TITLE
[doc] Fix 'decided' typo in coreFeatures.adoc

### PR DESCRIPTION
**Pull request by Inego**
Fix a typo

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@Inego Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/reactor?repositoryId=reactor/reactor-core&pullRequestId=2174)!

[Click here](https://cla.pivotal.io/sync/reactor?repositoryId=reactor/reactor-core&pullRequestId=2174) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by Inego** [comment]
@pivotal-issuemaster This is an Obvious Fix

**Comment by pivotal-issuemaster** [comment]
@Inego This Pull Request contains an obvious fix. Signing the Contributor License Agreement is not necessary.

File: docs/asciidoc/coreFeatures.adoc [hunk 1]
```diff
@@ -457,7 +457,7 @@ Flux.just(10)
 ----
 ====
 
-You also have the option of applying a `Predicate` on the exception to decided
+You also have the option of applying a `Predicate` on the exception to decide
 whether or not to recover, as the folloiwng example shows:
```
Comment (Inego) [inline docs/asciidoc/coreFeatures.adoc:461]: Aw man, I didn't notice a typo on the next line! `folloiwng`  :(

Comment (Inego) [reply docs/asciidoc/coreFeatures.adoc:461]: Could fix two typos in one sentence at once! Well, maybe next time...

Comment (simonbasle) [reply docs/asciidoc/coreFeatures.adoc:461]: ah snap XD
looks like a failed search-and-replace... maybe there are other occurrences in more files (including non-asciidoc ones)

Comment (simonbasle) [reply docs/asciidoc/coreFeatures.adoc:461]: ah sorry, misinterpreted the comment... distinct typo, got it

**Pull request merged**